test/distributed/org/apache/cassandra/distributed/test/accord/AccordIntegrationTest.java

```diff
@@ -18,55 +18,77 @@
 
-import static org.apache.cassandra.service.accord.db.AccordUpdate.UpdatePredicate.Type.*;
+import static org.apache.cassandra.service.accord.db.AccordUpdate.UpdatePredicate.Type.EQUAL;
+import static org.apache.cassandra.service.accord.db.AccordUpdate.UpdatePredicate.Type.NOT_EXISTS;
 
 public class AccordIntegrationTest extends TestBaseImpl
 {
     private static void assertRow(Cluster cluster, String query, int k, int c, int v)
     {
-        Object[][] result = cluster.coordinator(1).execute(query, ConsistencyLevel.QUORUM);
-        Assert.assertArrayEquals(new Object[]{new Object[] {k, c, v}}, result);
+        SimpleQueryResult result = cluster.coordinator(1).executeWithResult(query, ConsistencyLevel.QUORUM);
+        AssertUtils.assertRows(result, QueryResults.builder()
+                                                   .row(k, c, v)
+                                                   .build());
     }
 
     @Test
     public void testQuery() throws Throwable
     {
         String keyspace = "ks" + System.currentTimeMillis();
-        try (Cluster cluster = init(Cluster.build(2).start()))
+        try (Cluster cluster = init(Cluster.build(2).withConfig(c -> c.set("write_request_timeout_in_ms", TimeUnit.MINUTES.toMillis(10))).start()))
         {
             cluster.schemaChange("CREATE KEYSPACE " + keyspace + " WITH REPLICATION={'class':'SimpleStrategy', 'replication_factor': 2}");
             cluster.schemaChange("CREATE TABLE " + keyspace + ".tbl (k int, c int, v int, primary key (k, c))");
             cluster.forEach(node -> node.runOnInstance(() -> AccordService.instance.createEpochFromConfigUnsafe()));
             cluster.forEach(node -> node.runOnInstance(() -> AccordService.instance.setCacheSize(0)));
 
-            cluster.get(1).runOnInstance(() -> {
-                AccordTxnBuilder txnBuilder = new AccordTxnBuilder();
-                txnBuilder.withRead("SELECT * FROM " + keyspace + ".tbl WHERE k=0 AND c=0");
-                txnBuilder.withWrite("INSERT INTO " + keyspace + ".tbl (k, c, v) VALUES (0, 0, 1)");
-                txnBuilder.withCondition(keyspace, "tbl", 0, 0, NOT_EXISTS);
-                try
-                {
-                    AccordData result = (AccordData) AccordService.instance.node.coordinate(txnBuilder.build()).get();
-                    Assert.assertNotNull(result);
-                }
-                catch (InterruptedException | ExecutionException e)
-                {
-                    throw new AssertionError(e);
-                }
-            });
+            cluster.get(1).runOnInstance(() -> execute(txn()
+                                                       .withRead("SELECT * FROM " + keyspace + ".tbl WHERE k=0 AND c=0")
+                                                       .withWrite("INSERT INTO " + keyspace + ".tbl (k, c, v) VALUES (0, 0, 1)")
+                                                       .withCondition(keyspace, "tbl", 0, 0, NOT_EXISTS)));
```

Comment:
personal preference but felt this was cleaner than the same copy/paste, and lets me read the tx... can revert if you don't like.